content/id/docs/tasks/administer-cluster/reserve-compute-resources.md

```diff
@@ -0,0 +1,232 @@
+---
+title: Melakukan Reservasi Sumber Daya Komputasi untuk Daemon Sistem
+content_type: task
+min-kubernetes-server-version: 1.8
+---
+
+<!-- overview -->
+
+Node Kubernetes dapat dijadwalkan sesuai dengan kapasitas. Secara bawaan, Pod dapat menggunakan
+semua kapasitas yang tersedia pada sebuah Node. Ini merupakan masalah karena Node
+sebenarnya menjalankan beberapa _daemon_ sistem yang diperlukan oleh OS dan Kubernetes itu sendiri.
+Jika sumber daya pada Node tidak direservasi untuk _daemon-daemon_ tersebut, maka
+Pod dan _daemon_ akan berlomba-lomba menggunakan sumber daya yang tersedia, sehingga
+menyebabkan _starvation_ sumber daya pada Node.
+
+Fitur bernama `Allocatable` pada Node diekspos oleh kubelet yang berfungsi untuk melakukan
+reservasi sumber daya komputasi untuk _daemon_ sistem. Kubernetes merekomendasikan admin
+klaster untuk mengatur `Allocatable` pada Node berdasarkan tingkat kepadatan (_density_) beban kerja setiap Node.
+
+
+
+## {{% heading "prerequisites" %}}
+
+
+{{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
+Kamu harus menjalankan Kubernetes server dengan versi 1.17 atau yang lebih baru
+untuk menggunakan perintah baris kubelet dengan opsi `--reserved-cpus` untuk
+menyetel [daftar reservasi CPU secara eksplisit](#daftar-reservasi-CPU-secara-eksplisit).
```

Comment:
Link not working, it should be...
```suggestion
menyetel [daftar reservasi CPU secara eksplisit](#melakukan-reservasi-daftar-cpu-secara-eksplisit).
```
or just make a custom ID for that section to match the one used here...